js/private/nodejs_binary.bzl

```diff
@@ -144,13 +144,17 @@ def _bash_launcher(ctx, linkable, args):
         ])
     else:
         node_path = ""
-    node_path = "export NODE_PATH=" + ":".join(node_paths)
+    node_path = ":".join(node_paths)
     ctx.actions.write(
         launcher,
         """#!{bash}
 {rlocation_function}
 set -o pipefail -o errexit -o nounset
-{node_path}
+if [[ -n "${{NODE_PATH-}}" ]]; then
```

Comment:
does there need to be a corresponding fix in the windows cmd.bat version?

Reply:
Yes, we will need that. I'll make another pass on whole thing